spec/services/proofing/lexis_nexis/instant_verify/verification_request_spec.rb

```diff
@@ -98,4 +98,15 @@
       end
     end
   end
+
+  describe '#formatted_address' do
+    context 'when ZIP code is nil' do
+      it 'returns nil without raising an exception' do
+        applicant[:zipcode] = nil
+
+        expect { subject.send(:formatted_address) }.not_to raise_error(NoMethodError)
+        expect(subject.send(:formatted_address)['Zip5']).to be_nil
```

Comment:
`nil.to_s == ''` so either we need `&.to_s` or this needs to be `''`
```suggestion
        expect(subject.send(:formatted_address)['Zip5']).to eq('')
```

Reply:
I'd also consider using `#fetch` because I think we have a string vs symbol mismatch here and `#fetch` will yell loudly when the key is not there
```suggestion
        expect(subject.send(:formatted_address).fetch(:Zip5)).to eq('')
```

Reply:
I am game to change this but don't 100% follow your suggestion. As implemented, nil is correct here, because we never get to the `to_s` if the nil-safe operator intervenes.

Or are you saying we should do both--add the `&.to_s` bit and then update this to expect an empty string?

Reply:
Oh! Nevermind, you're right.